### ISSUE TITLE
drush vget failing with v1.12.1

### ISSUE THREAD

**Issue by sepal** [drud/ddev#2038]
**Describe the bug**
After upgrading to v1.12.1 executing `drush vget` for drupal 7 throws the following warnings & error:

```
fopen(phar:///usr/local/bin/drush8/vendor/composer/../../src/Internal/Symfony/Yaml/Dumper.php): failed to open stream: phar error: "src/Internal/Symfony/Yaml/Dumper.php" is not a file in phar "/usr/local/bin/drush8" PharStreamWrapper.php:487                                                                                                                [warning]
include(phar:///usr/local/bin/drush8/vendor/composer/../../src/Internal/Symfony/Yaml/Dumper.php): failed to open stream: "TYPO3\PharStreamWrapper\PharStreamWrapper::stream_open" call failed ClassLoader.php:444                                                                                                                                                [warning]
include(): Failed opening 'phar:///usr/local/bin/drush8/vendor/composer/../../src/Internal/Symfony/Yaml/Dumper.php' for inclusion (include_path='.:/usr/share/php') ClassLoader.php:444                                                                                                                                                                          [warning]
Drush command terminated abnormally due to an unrecoverable error.                                                                                                                                                                                                                                                                                               [error]
```

**Comment by rfay** [comment]
I can recreate that inside the container with v1.12.1

It seems to be a bug in drush 8.3.1, AFAICT. We upgraded to 8.3.2 (which doesn't display the problem) in https://github.com/drud/ddev/pull/2000

Please try the edge ddev version, v1.13.0-alpha1, I don't think you'll have any trouble with it.  https://github.com/drud/ddev/releases/tag/v1.13.0-alpha1  (If you use homebrew/linuxbrew, `brew tap drud/ddev-edge/ddev && brew upgrade drud/ddev-edge/ddev`

**Comment by sepal** [comment]
Hmm, ` brew tap drud/ddev-edge/ddev` gives me the following error:

```sh
Error: Invalid tap name 'drud/ddev-edge/ddev'
```

Would it also be possible to somehow override the webserver Dockerfile while still using ddev 1.12?

**Comment by sepal** [comment]
As a temporary fix, we just override the the drush version. :P
```
ddev exec -s web sudo "curl -sSL "https://github.com/drush-ops/drush/releases/download/8.3.2/drush.phar" -o /usr/local/bin/drush8 && sudo chmod +x /usr/local/bin/drush8"
```

**Comment by rfay** [comment]
Sorry, I should have said `brew install drud/ddev-edge/ddev`, but the [release has full info](https://github.com/drud/ddev/releases/tag/v1.13.0-alpha2). Also, alpha2 is out now, would love to have you use it. To make things easy, `brew untap drud/ddev && brew tap drud/ddev-edge && brew upgrade ddev`. I assume that upgrading drush did the job for you though.

You can also add your drush upgrade as a post-start hook or (better) in custom web-build/Dockerfile.

**Comment by rfay** [comment]
Closing this as seems to be solved by drush, already done in #2000, but happy to continue the conversation here.
